### PR TITLE
Feature/holo 631 transition indexer to gql

### PR DESCRIPTION
**Pull request by alexanderattar**
## Describe Changes

- Update Indexer to use GraphQL instead of REST for all endpoints

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

File: src/commands/indexer/index.ts [hunk 1]
```diff
@@ -1428,11 +1477,21 @@
...
+
+    const query = gql`
+    query($contractAddress: String!, $tokenId: String!) {
+      nftByContractAddressAndTokenId(contractAddress: $contractAddress, tokenId: $tokenId) {
+        id
+        contractAddress
+        tokenId
+      }
+    }
+    `
```
Comment (sogoiii) [inline src/commands/indexer/index.ts:1532]: i dont think this method is being used. should this be kpet here?

Comment (alexanderattar) [reply src/commands/indexer/index.ts:1532]: I'm not 100% I know what you're referring to. The `query` is passed along into the db job for the callback via `updateCrossChainTransactionCallback` which is where the the actual API call is made. Let me know if it was something else I'm missing though

Comment (sogoiii) [reply src/commands/indexer/index.ts:1532]: @alexanderattar ok my bad. I mean to put that comment on the function `async sendMutationRequest(options: PatchOptions, tags: (string | number)[]): Promise<void>` on line 1518. When i dont believe this method is used anywhere. I think that method was moved over to `apiService.sendMutationRequest`.

**Review by sogoiii — APPROVED** [review]
👍

**Pull request merged**